src/js/components/Poll/PublicRegisterModal.vue

```diff
@@ -149,7 +149,7 @@ export default {
 				}
 			}
 			return {
-				result: t('polls', '{username} is valid.', { username: this.userName }),
+				result: t('polls', 'Name {username} is valid.', { username: this.userName }),
```

Comment:
Sounds strange (Name Peter is valid). Term should be kept short.

Reply:
You're right 😉. Changed in #1573.